.github/workflows/signable-rspec.yml

```diff
@@ -0,0 +1,34 @@
+name: CI RSpec Tests
+
+on: [push, pull_request]
+  push:
+    paths:
+    - 'signable/**'
+  pull_request:
+    paths:
+    - 'signable/**'
+
+jobs:
+  build:
+    name: CI
+    runs-on: ubuntu-latest
+    env:
+      api-dir: ./signable
+
+steps:
+  - uses: actions/checkout@v1
+  - uses: actions/setup-ruby@v1
+    with:
+      ruby-version: 2.3.0
```

Comment:
Seeing this. Would it be good to bump the ruby version soon? (Not this pr 😉 )

Reply:
Feel free to raise a ticket for it.